app/lib/definitions.ts

```diff
@@ -30,7 +30,7 @@ export type Claim = {
 export type OriginalPresentationEventMessage = {
   ref: string;
   claims?: OriginalClaim[];
-  status: "refused" | "ok" | "connected";
+  status: "refused" | "ok" | "connected" | "no-compatible-credentials";
```

Comment:
If you look at [Service Agent documentation](https://github.com/2060-io/2060-service-agent/blob/main/doc/service-agent-api.md#presentation-callback-api), you'll see that there are other possible status, namely 'unspecified-error' and 'verification-error'. Could you also handle them?

I think it would be a good idea to log somewhere if an unknown status is received. This way it will be easier to debug in case new statuses are added into Service Agent API and we don't know what's going on.